src/App.js

```diff
@@ -1,7 +1,10 @@
+import Store from "./storage/store";
+
 export class App {
   constructor() {
     this.scene = null;
     this.quality = "low";
+    this.store = new Store();
```

Comment:
should we try to make this a singleton that is passed in? we currently create one at the root of `hub.js`